packages/editor/src/client/routes/commentList.tsx

```diff
@@ -0,0 +1,408 @@
+  switch (columnField) {
+    case 'createdAt':
+      return CommentSort.CreatedAt
+    case 'modifiedAt':
+      return CommentSort.ModifiedAt
+    default:
+      return null
+  }
+}
+
+export function CommentList() {
+  const {t} = useTranslation()
+  const [page, setPage] = useState(1)
+  const [limit, setLimit] = useState(10)
+  const [sortField, setSortField] = useState('createdAt')
+  const [sortOrder, setSortOrder] = useState<'asc' | 'desc'>('desc')
+
+  const [filter, setFilter] = useState('')
+
+  const [comments, setComments] = useState<CommentRefFragment[]>([])
+
+  const [approveComment, {loading: isApproving}] = useApproveCommentMutation()
+  const [requestChanges, {loading: isRequestingChanges}] = useRequestChangesOnCommentMutation()
+  const [rejectComment, {loading: isRejecting}] = useRejectCommentMutation()
+
+  const {data, refetch, loading: isLoading} = useCommentListQuery({
+    variables: {
+      first: limit,
+      skip: page - 1,
+      sort: mapColumFieldToGraphQLField(sortField),
+      order: mapTableSortTypeToGraphQLSortOrder(sortOrder)
+    },
+    fetchPolicy: 'network-only'
+  })
+
+  useEffect(() => {
+    refetch({
+      first: limit,
+      skip: page - 1,
+      sort: mapColumFieldToGraphQLField(sortField),
+      order: mapTableSortTypeToGraphQLSortOrder(sortOrder)
+    })
+  }, [filter, page, limit, sortOrder, sortField])
+
+  useEffect(() => {
+    if (data?.comments?.nodes) {
+      setComments(data.comments.nodes)
+      if (data.comments.totalCount + 9 < page * limit) {
+        setPage(1)
+      }
+    }
+  }, [data?.comments])
+
+  const [isConfirmationDialogOpen, setConfirmationDialogOpen] = useState(false)
+  const [currentComment, setCurrentComment] = useState<Comment>()
+  const [confirmAction, setConfirmAction] = useState<ConfirmAction>()
+  const [rejectionReason, setRejectionReason] = useState<CommentRejectionReason>()
+
+  const mapModalTitle = () => {
+    let title: string
+    switch (confirmAction) {
+      case ConfirmAction.Approve:
+        title = t('comments.panels.approveComment')
+        break
+      case ConfirmAction.Reject:
+        title = t('comments.panels.rejectComment')
+        break
+      case ConfirmAction.RequestChanges:
+        title = t('comments.panels.requestChangesOnComment')
+        break
+      default:
+        title = ''
+        break
+    }
+    return title
+  }
+
+  return (
+    <>
+      <FlexboxGrid>
+        <FlexboxGrid.Item colspan={16}>
+          <h2>{t('comments.overview.comments')}</h2>
+        </FlexboxGrid.Item>
+        <FlexboxGrid.Item colspan={24} style={{marginTop: '20px'}}>
+          <InputGroup>
+            <Input value={filter} onChange={value => setFilter(value)} />
+            <InputGroup.Addon>
+              <Icon icon="search" />
+            </InputGroup.Addon>
+          </InputGroup>
+        </FlexboxGrid.Item>
+      </FlexboxGrid>
+
+      <Table
```

Comment:
Checkout https://github.com/wepublish/wepublish/pull/172/files#diff-6efbad89c5a8f1d9efb955a581e41cc41b43726e032b3aa53a2f67bfc3437a69R106 

with the extra div and the min height you can avoid a bug where the autoHeight sometimes cuts of the last entry.